src/users/users.service.ts

```diff
@@ -96,9 +145,10 @@ export class UsersService {
             if (createuserDto.managerId != undefined) {
                 let Manager = await this.usersRepository.findOne({where:{companyId: createuserDto.companyId , 
                     employeeId : createuserDto.managerId}});
-                // If manager status of managerId is false, then set it to true
+                // If manager status of managerId is false, then set it to true and set role to Admin
                 if (Manager != undefined && Manager.isManager == false) {
                     Manager.isManager = true;
+                    Manager.role = Role.Admin;
```

Comment:
I remember there being some discussion of having a default admin role, but I don't think we want to assign managers to be admins? That is, this function, on inserting one employee, could change the role of another employee, and I don't think we want to do that